File: js/lista-3-facul/questao-2.js
// 2) Leia 20 valores reais e calcule seu somatório utilizando a instrução do...while
const prompt = require('prompt-sync')({ sigint: true });

let soma = 0;
let index = 1;

do {
  const entrada = Number(prompt(`Digite a entrada ${index}: `));

  if (!isNaN(entrada)) {
    soma += entrada;
    index += 1;
  } else {
    console.log('Digite apenas números.');
  }
} while (index <= 20);

console.log(`A soma é: ${soma}`);

File: python/lista-3-facul/questao-2.py
soma = 0;
index = 1;

# I'll use a flag as there is no do while in python
while True:
  entrada = input(f'Digite a entrada {index}: ');

  if entrada.isnumeric():
    soma += int(entrada);
    index += 1;
  else:
    print('Digite apenas números.');

  if index >= 20:
    break;

print(f'\nA soma é: {soma}');
